lambdas/src/apis/collections/routes.ts

```diff
@@ -12,11 +12,14 @@ export function initializeCollectionsRoutes(
   router: Router,
   client: SmartContentClient,
   theGraphClient: TheGraphClient,
-  offChainManager: OffChainWearablesManager
+  offChainManager: OffChainWearablesManager,
+  rootStorageLocation: string
 ): Router {
   router.get('/standard/erc721/:chainId/:contract/:option/:emission?', createHandler(client, getStandardErc721))
-  router.get('/contents/:urn/image', createHandler(client, contentsImage))
-  router.get('/contents/:urn/thumbnail', createHandler(client, contentsThumbnail))
+  router.get('/contents/:urn/image', createHandler(client, contentsImage, rootStorageLocation))
+  router.get('/contents/:urn/image/:size', createHandler(client, contentsImage, rootStorageLocation))
+  router.get('/contents/:urn/thumbnail', createHandler(client, contentsThumbnail, rootStorageLocation))
```

Comment:
Are routes `'/contents/:urn/image'` and `'/contents/:urn/thumbnail'` implemented?

Reply:
yes, same function is used both for `'/contents/:urn/image'` and `'/contents/:urn/image/:size'` 

(same for thumbnail)